css/home.css

```diff
@@ -0,0 +1,487 @@
+  margin-bottom: 25px;
+}
+
+.date {
+  color: #272a31;
+  font-size: 25px;
+  font-weight: 500;
+  padding-bottom: 10px;
+}
+
+.venue {
+  color: #272a31;
+  font-size: 20px;
+  line-height: 25px;
+  font-weight: 500;
+}
+
+.prog {
+  background-color: #272a31;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.prog h2 {
+  color: #fff;
+  font-size: 28px;
+  font-weight: 600;
+  margin: 30px 0 15px;
+}
+
+.prog .prog-line {
+  border: 1px solid #ec5242;
+  width: 50px;
+  margin-bottom: 40px;
+}
+
+.prog .row-container {
+  list-style: none;
+}
+
+.prog .row-container li {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  background-color: rgba(255, 255, 255, 0.1);
+  padding: 20px 5px;
+  margin-bottom: 10px;
+}
+
+.row-container img {
+  width: 28%;
+  padding-right: 5px;
+}
+
+.speaker-info h3 {
+  font-size: 25px;
+}
+
+.row-container li h3 {
+  color: #ec5242;
+  font-size: 20px;
+  padding-right: 29px;
+}
+
+.row-container p {
+  width: 60%;
+  color: #fff;
+  font-size: 16px;
+  padding-left: 30px;
+}
+
+.prog .join-program {
+  background-color: #ec5242;
+  color: #fff;
+  padding: 35px 55px;
+  margin: 40px 0;
+  font-size: 20px;
+  border: 0;
+  cursor: pointer;
+}
+
+.prog button:hover {
+  color: #ec5242;
+  background-color: #f4f4f4;
+}
+
+/* ======== Dynamic Section ======= */
+
+.talks {
+  padding: 20px;
+  margin-bottom: 40px;
+}
+
+.subheading {
+  font-weight: 700;
+  font-size: 25px;
+  width: fit-content;
+  margin: 30px auto 15px auto;
+}
+
+.talks .prog-line {
+  width: 50px;
+  margin: auto;
+  margin-bottom: 40px;
+  border: 1px solid #ec5242;
+}
+
+.grid-container .speaker {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  margin-bottom: 50px;
+}
+
+.footer img {
+  width: 50%;
+  padding: 0 5%;
+}
+
+.pic-container {
+  width: 80%;
+  max-width: 150px;
+  margin-right: 20px;
+  height: 150px;
+  position: relative;
+  overflow-y: hidden;
+}
+
+.pic-container .image-container {
+  width: 90%;
+  height: 90%;
+  position: relative;
+  top: 20px;
+  left: 20px;
+}
+
+.pic-container .image-container img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  object-position: center;
+  position: absolute;
+}
+
+.talks .square {
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: block;
+  background-color: #272a31;
+  width: 70px;
+  height: 70px;
+  z-index: -1;
+}
+
+.speaker-info {
+  max-width: 50%;
+}
+
+.speaker-info h4 {
+  font-weight: 400;
+  font-size: 17px;
+  color: #ec5242;
+  font-style: italic;
+  margin: 10px 0;
+}
+
+.speaker-info .line-dark {
+  width: 30px;
+  margin: 10px 0 10px 0;
+  border: 1px solid #d3d3d3;
+}
+
+.speaker-info p {
+  color: #272a31;
+  font-size: 16px;
+  line-height: 20px;
+}
+
+.talks .more {
+  display: none;
+}
+
+#partners {
+  display: none;
+}
+
+.footer {
+  display: flex;
+  align-items: center;
+  padding: 1rem 5%;
+  background: #272a31da;
```

Comment:
- [Optional] Hi @krisbobo, based on the requirements of this project, the footer of the mobile size is supposed to have a white background and not a sort of transparent one. You may kindly ameliorate this.